app/pages/patientdata/patientdata.js

```diff
@@ -177,38 +178,37 @@ export let PatientData = React.createClass({
     };
 
     if (this.props.isUserPatient) {
-      
       content = (
-        <div className="patient-data-message-no-data">
-          <p>{'There is no data in here yet!'}</p>
-          <a
-            href={this.props.uploadUrl}
-            target="_blank"
-            onClick={handleClickUpload}>Upload data</a>
-          <p>
-            {'Or try '}<a href="" onClick={this.handleClickNoDataRefresh}>refreshing</a>{' the page.'}
+        <div className="patient-data-uploader-message">
+          <h1>To see your data, you’ll need to upload it!</h1>
+          <UploaderButton
+            buttonUrl='http://chrome.google.com/webstore/detail/tidepool-uploader/cabklgajffclbljkhmjphejemhpbghfb'
```

Comment:
Could we pull these URLs out as constants or perhaps defaultProps for easier access in future in case they need to be updated?